### ISSUE TITLE
Bundling angular rc5 with ngUpgrade breaks after 0.15.26

### ISSUE THREAD

**Issue by guojenman** [systemjs/builder#675]
**I'm reporting a bug**

bundling with version 0.15.26, it works
bundling with version 0.15.27 and above, it does not work

The bundling completes with no error, but when trying to run the code in the browser, bundling with 0.15.27 or greater gives the error of **Error: No provider for ViewUtils!**

**Steps to reproduce**
npm i
npm run serve
(open in browser & will see error in console)

npm i systemjs-builder@0.15.26
npm run serve
(open in browser & will work just fine)

**attached file:** [ng2bad-build.zip](https://github.com/systemjs/builder/files/435651/ng2bad-build.zip)

One thing I noticed is that bundling without ngUpgrade is fine, but since I'm working on a mixed mode app, ngUpgrade is a requirement. Should this be reported as a bug to the angular team?

**Comment by jrauschenbusch** [comment]
Hi @guojenman. I have reproduced your bug. From v0.15.27 to v0.15.29 your example code doen't work. But it works with v0.15.30. The cause is related to the babel plugin used by the new cjs transformer, which was introduced in v0.15.27. It had some bugs (https://github.com/systemjs/babel-plugin-transform-cjs-system-wrapper/issues/8. https://github.com/systemjs/builder/issues/673) which were solved in the subsequent releases.
